app/models/data_update_script.rb

```diff
@@ -25,6 +25,40 @@ def scripts_to_run?
       false
     end
 
+    def force_run(id)
+      script = DataUpdateScript.find(id)
+      log_status(script)
+      run_script(script)
+
+      script
+    end
+
+    def run_script(script)
+      require script.file_path
+      script.file_class.new.run
+
+      script.mark_as_finished!
+      log_status(script)
+    rescue StandardError => e
+      script.mark_as_failed!(e)
+      log_status(script)
+
+      Honeybadger.notify(e, context: { script_id: script.id })
+    end
+
+    def log_status(script)
+      status = script.status
+      file_name = script.file_name
+
+      logger_destination = status.to_sym == :failed ? :error : :info
+      Rails.logger.public_send(
+        logger_destination,
+        "time=#{Time.current.rfc3339}, script=#{file_name}, status=#{status}",
+      )
+
+      DatadogStatsClient.increment("data_update_scripts.status", tags: ["status:#{status}", "script_name:#{file_name}"])
+    end
```

Comment:
As you mentioned privately, these contain business logic that should probably be splint in a service that's called in the controller and the `DataUpdateWorker`